src/audio/src/src.c

```diff
@@ -988,7 +988,7 @@ static int src_init(struct processing_module *mod)
 	comp_dbg(dev, "src_init()");
 
 	/* validate init data - either SRC sink or source rate must be set */
-	if (src_rate_check(cfg->init_data) < 0) {
+	if (!cfg->init_data || src_rate_check(cfg->init_data) < 0) {
```

Comment:
and `cfg->size == sizeof(*cd->ipc_config)`

Reply:
Added.